src/js/game.js

```diff
@@ -0,0 +1,195 @@
+import { ERROR_MESSAGE, CONGRATS_MESSAGE, INPUT_TYPE } from "./constants.js";
+import { $ } from "./utils.js";
+
+export default function CarGame() {
+  const form = $("form");
+  const names = $("#names");
+  const namesInput = $("#names input");
+  const nameCards = $("#nameCards");
+  const nameSubmitBtn = names.querySelector("button");
+  const gameWinner = $("#winner");
+
+  let carNames = [];
+  let times = "";
+  let carsDistance = [];
+  let winner = [];
+
+  namesInput.addEventListener("keyup", ({ target }) => {
+    carNames = target.value;
+    carNames = checkCarNames(carNames);
+  });
+
+  function onCarNamesSubmit() {
+    nameSubmitBtn.addEventListener("click", () => {
+      showGameTimesInput();
+      onGameTimesSubmit();
+      preventInput(INPUT_TYPE.NAME);
+    });
+  }
+
+  function checkCarNames(names) {
+    const namesArr = names.split(",");
+    const nameBlankCheck = namesArr
+      .map(name => name.replace(/(^\s*)|(\s*$)/gi, ""))
```

Comment:
이름을 공백으로 입력했는지 체크하는 로직인가요?
[trim()](https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) 함수를 적용해봐도 좋을 것 같습니다~
```suggestion
      .map(name => name.trim())
```